### ISSUE TITLE
comma-spacing issue

### ISSUE THREAD

**Issue by Globegitter** [eslint/eslint#2250]
With comma-spacing on I am having issues with following array declaration:

```js
patterns: [
          path.resolve(sails.config.appPath,'api'),
          path.resolve(sails.config.appPath,'config')
        ]
```

I get the following errors:

```
index.js
  48:43  error  A space is required after ','  comma-spacing
  48:52  error  Trailing spaces not allowed    no-trailing-spaces //this is only if I actually add a space after the comma
  49:43  error  A space is required after ','  comma-spacing
```

So how is one to have `comma-spacing` on but then also have an array declared over multiple lines if each element is just very long like in this case?

**Comment by nzakas** [comment]
Before we look in, can you please provide us with some information:

1. What version of ESLint are you using?
1. Are you using the default parser or something else?
1. What does your configuration for `comma-spacing` look like?

Thanks.

**Comment by lo1tuma** [comment]
Looks like the space is missing after the comma of the first argument of the function call.

**Comment by Globegitter** [comment]
Oh yes, @lo1tuma thanks. Sorry somehow I was blind to the numbers it already showed me.
